crates/holochain_p2p/src/types/actor.rs

```diff
@@ -203,9 +203,30 @@ impl Default for GetActivityOptions {
 
 /// Trait defining the main holochain_p2p interface.
 pub trait HcP2p: 'static + Send + Sync + std::fmt::Debug {
+    /// Test access to underlying kitsune instance.
+    #[cfg(feature = "test_utils")]
+    fn test_kitsune(&self) -> &kitsune2_api::DynKitsune;
+
     /// Test utility to force local agents to report full storage arcs.
     #[cfg(feature = "test_utils")]
-    fn test_set_full_arcs(&self, space: kitsune2_api::SpaceId) -> BoxFut<'_, ()>;
+    fn test_set_full_arcs(&self, space: kitsune2_api::SpaceId) -> BoxFut<'_, ()> {
+        Box::pin(async {
+            for agent in self
+                .test_kitsune()
+                .space(space)
+                .await
+                .unwrap()
+                .local_agent_store()
+                .get_all()
+                .await
+                .unwrap()
+            {
+                agent.set_cur_storage_arc(kitsune2_api::DhtArc::FULL);
+                agent.set_tgt_storage_arc_hint(kitsune2_api::DhtArc::FULL);
+                agent.invoke_cb();
```

Comment:
What's the purpose of this `invoke_cb()` here. Do we know what callback had been registered that's now being invoked here and may that matter?

Reply:
I see now that this code is not actually new as of this PR but just had been moved here to become a default implementation for that method. I'm still curious though about the `invoke_cb`.